api/BUILD

```diff
@@ -28,10 +23,7 @@ string_flag(
 cc_library(
     name = "api",
     hdrs = glob(["include/**/*.h"]),
-    defines = select({
-        ":with_external_abseil": ["HAVE_ABSEIL"],
-        "//conditions:default": [],
-    }) + select({
+    defines = ["HAVE_ABSEIL"] + select({
```

Comment:
Does it mean we won't be able to build the otel-cpp with the internal abseil snapshot?

Reply:
It appears that we now always use the external abseil  or stl version of `variant` type, and not the internal snapshot with bazel. This should be fine, if we can document it in (probably) [INSTALL.md](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/INSTALL.md#build-instructions-using-bazel)

Reply:
this is covered in dependencies.md fwiw

Reply:
INSTALL.md probably needs more cleanup and revision, as it seem outdated.

I suggest to address it in a separate PR, to avoid delaying this change.